project/Build.scala

```diff
@@ -21,11 +21,16 @@ import org.scalastyle.sbt.ScalastylePlugin
 
 object reactiveneo extends Build {
 
-  val UtilVersion = "0.4.0"
-  val ScalatestVersion = "2.2.0-M1"
-  val FinagleVersion = "6.20.0"
-  val playVersion = "2.3.4"
+  val scalaMajorVersion = "2.11"
+  val scalaMinorVersion = "6"
+  val scalaFullVersion = scalaMajorVersion + "." + scalaMinorVersion
+
+  val UtilVersion = "0.5.0"
```

Comment:
Currently this version is missing from our Artifactory due to cache issues we had last week. You may want to upgrade to something like 0.6.12 I believe. The phantom feature/connectors branch is using the latest.